textacy/tm/topic_model.py

```diff
@@ -103,9 +104,13 @@ class TopicModel(object):
         - http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.TruncatedSVD.html
     """
 
+    _required_trained_model_attr = {"transform", "components_", "n_topics"}
+
     def __init__(self, model, n_topics=10, **kwargs):
         if isinstance(model, (NMF, LatentDirichletAllocation, TruncatedSVD)):
             self.model = model
+        elif all([hasattr(model, required_attr) for required_attr in self._required_trained_model_attr]):
```

Comment:
nit: you can drop the brackets on the list (`[` and `]`)

Reply:
addressed